frontend/src/components/PublicLayout.tsx

```diff
@@ -235,6 +235,18 @@ export default function PersistentDrawerLeft(props) {
 						</ListItemButton>
 					</ListItem>
 				</List>
+				<Divider/>
+				<List>
+					<ListItem key={"public_search"} disablePadding>
+						<ListItemButton component={RouterLink} to="/public_search">
+							<ListItemIcon>
+								<SearchDatasetIcon />
```

Comment:
It gives error here as you never imported SearchDatasetIcon in this file.

Reply:
@tcnichol is this addressed?

Reply:
Just pushed a fix for this.